posthog/management/commands/migrate_clickhouse_consumer_offsets.py

```diff
@@ -0,0 +1,109 @@
+import sys
+import time
+
+import structlog
+from django.core.management.base import BaseCommand
+from kafka import KafkaAdminClient
+
+from posthog.kafka_client.client import build_kafka_consumer
+
+logger = structlog.get_logger(__name__)
+
+
+CLICKHOUSE_CONSUMER_GROUP = "group1"
+TOPICS_TO_MIGRATE = [("clickhouse_app_metrics", "clickhouse-inserter-clickhouse_app_metrics")]
+
+
+class Command(BaseCommand):
+    help = "Migrate ClickHouse consumer group offsets."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--fake", action="store_true", help="Mark migrations as run without actually running them.")
+        parser.add_argument(
+            "--check", action="store_true", help="Mark migrations as run without actually running them."
+        )
+
+    def handle(self, *args, **options):
+        admin = KafkaAdminClient()
+
+        for topic, group in TOPICS_TO_MIGRATE:
```

Comment:
nit: for a second wave of topic migrations, we won't want to re-migrate `clickhouse_app_metrics` again and rewind, so that should be a command param. But we can change that later on.